src/canmatrix/formats/dbc.py

```diff
@@ -142,7 +145,8 @@ def dump(mydb, f, **options):
 
     # free signals are in special frame in dbc...
     if len(db.signals) > 0:
-        free_signals_dummy_frame = canmatrix.Frame("VECTOR__INDEPENDENT_SIG_MSG", id = canmatrix.ArbitrationId(0x40000000, extended=True))
```

Comment:
The converter `ArbitrationId.from_compound_integer` doesn't accept ArbitrationId itself. Id must be assigned separately.
But we need to change the converter anyway, because the classmethod is not supported by the type check.